src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -120,6 +120,20 @@ static bool _parseNumber(const char** content, float* number)
 }
 
 
+//custom strndup() for portability
+char* _strndup(const char* str, size_t n)
```

Comment:
This method has been added in tvgXmlParser.cpp.
I think it's time to refactor code. (move it to tvgSvgUtil.cpp) and use it here.